app/code/Magento/Ui/view/base/web/js/lib/knockout/bindings/scope.js

```diff
@@ -13,6 +13,43 @@ define([
 ], function (ko, registry, $t, renderer, $, consoleLogger) {
     'use strict';
 
+    /**
+     * Check if an element is visible in the viewport
+     *
+     * @param {HTMLElement} el
+     * @returns bool
+     */
+    function isInViewport(el) {
+        if (!el.checkVisibility()) return false;
```

Comment:
@rogerdz 
Possible issue like the one described with similar condidtion check for `lib/web/mage/apply/main.js`